packages/expo-dev-launcher/plugin/src/pluginConfig.ts

```diff
@@ -36,6 +36,11 @@ export type PluginConfigOptions = {
    * @default 'launcher'
    */
   launchModeExperimental?: 'most-recent' | 'launcher';
+  /**
+   * Determines whether to add the `exp+slug` URL scheme or not.
+   * @default 'true'
+   */
+  schemeEnabled?: boolean;
```

Comment:
naming things is hard so feel free to suggest a more appropriate name for this...

Reply:
`generatedSchemeEnabled` perhaps?

Reply:
addressed